api/health.go

```diff
@@ -285,8 +288,54 @@ func normalizeProperty(unitName string, p map[string]interface{}, d DCOSHelper)
 	}
 }
 
+func updateSystemMetrics() (sysMetrics sysMetrics, err error) {
```

Comment:
^^ Not a blocker, just something to consider without taking into account the overall structure of implementation.